src/sentry/utils/snuba.py

```diff
@@ -556,12 +557,20 @@ def snuba_query(params):
         except urllib3.exceptions.HTTPError as err:
             raise SnubaError(err)
 
-    if len(snuba_param_list) > 1:
-        query_results = _query_thread_pool.map(snuba_query, query_param_list)
-    else:
-        # No need to submit to the thread pool if we're just performing a
-        # single query
-        query_results = [snuba_query(query_param_list[0])]
+    with sentry_sdk.start_span(
+        op="start_snuba_query", description="running {} snuba queries".format(len(snuba_param_list))
```

Comment:
Any chance of unicode bytes being in your param list?

Reply:
Maybe, but this is just taking the `len` of the list. Regardless, we should likely use `u` here.